src/components/home/MemberExperienceTitle.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+const MemberTitle = () => {
+  return (
+    <div className="flex flex-col">
```

Comment:
```suggestion
    <div">
```
remove these classes, they don't do anything